spec/models/user_spec.rb

```diff
@@ -61,7 +61,10 @@
 
       describe "with many organisations" do
         let(:organisations) { [organisation, create(:organisation)] }
-        it { expect { subject }.not_to change(user, :organisation_ids) }
+        it "should not change organisations" do
+          subject
+          expect(organisations).to match_array(user.organisations)
```

Comment:
Je pense que ton fix est bien mais je ne comprends pas le test en fait tu peux m'aider ?

Je comprends que ça veut tester ce cas : 
- user1 est associé à orga1 et orga2
- on fait user1.add_orga(orga2)
- on vérifie que ça n'a pas changé user1.organisations

MAIS 
dans la manière dont est écrit le test, je n'ai pas l'impression que ça fasse ça.  Car je ne vois pas comment les deux lignes:
`let(:organisation) { create(:organisation) }`
`let(:organisations) { [organisation, create(:organisation)] }`
pointent vers la même orga (orga2) ?

Reply:
On regarde si en ajoutant une orga à un utilisateur alors qu'elle est deja associé, ça ne change rien.

Reply:
organisations = [orga1, orga2]
user.organisations = organisations
user.add_organisation(orga1)
expect(user.organisations) to eq (organisations)

Reply:
got it merci !